docs/MapColonies/3D/Guides/3D_getting_started.md

```diff
@@ -60,7 +60,7 @@ body (XML):
                     </ogc:PropertyIsEqualTo>
```

Comment:
Please remove `classification` from this query.

Reply:
done